bundles/org.eclipse.equinox.servletbridge/src/org/eclipse/equinox/servletbridge/FrameworkLauncher.java

```diff
@@ -46,6 +46,7 @@ public class FrameworkLauncher {
 	private static final String REFERENCE_SCHEME = "reference:"; //$NON-NLS-1$
 	private static final String CONFIG_INI = "config.ini"; //$NON-NLS-1$
 	private static final String DOT_JAR = ".jar"; //$NON-NLS-1$
+	private static final String[] VERSION_SEPS = {"_", "-" }; //$NON-NLS-1$; //$NON-NLS-2$ 
```

Comment:
Instead of an array here I would rather see two separately named constants like `VERSION_SEP_UNDERSCORE` and `VERION_SEP_DASH`.